packages/react-rest/src/services/hatchifyReactRest/hatchifyReactRest.ts

```diff
@@ -46,7 +46,11 @@ export type ReactRest<Schema extends SchemaRecord> = {
     useDeleteOne: () => [(id: string) => void, Meta]
     useAll: (query?: QueryList) => [Record[], Meta]
     useOne: (query: QueryOne | string) => [Record | undefined, Meta]
-    useUpdateOne: (id: string) => [(data: CreateData) => void, Meta, Record?]
+    useUpdateOne: () => [
+      (data: UpdateData) => void,
+      Meta,
+      Record | undefined | null,
```

Comment:
the backend returns null on patch requests, this is gonna change in the future, but for now (and other backends that do this), we pass through that null to the frontend